### PR TITLE
INS-228-Dania-Khan

### PR DESCRIPTION
**Pull request by daniakhanpurdue**
Moved merge.adoc from old examples book to new one for R

### CODE REVIEW

File: content/modules/r/pages/merge.adoc [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+[source, R]
+----
+head(myDF)
+----
+
+It is harder to tell which column in the `airportsDF` gives the 3-letter codes, but these are the iata codes.
+
+[source, R]
+----
+head(airportsDF)
```
Comment (lacykq) [inline content/modules/r/pages/merge.adoc:86]: Output for the code snippet?

File: content/modules/r/pages/merge.adoc [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+----
+
+It is perhaps easier to see this from the tail of airportsDF:
+
+[source, R]
+----
+tail(airportsDF)
```
Comment (lacykq) [inline content/modules/r/pages/merge.adoc:93]: Add the output for the code snippet?

File: content/modules/r/pages/merge.adoc [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+mynewDF <- merge(myDF, airportsDF, by.x="Origin", by.y="iata")
+----
+
+The resulting data frame has the same size as `myDF`:
+
+[source, R]
+----
+dim(myDF)
+dim(mynewDF)
+----
```
Comment (lacykq) [inline content/modules/r/pages/merge.adoc:109]: Add the output for the code snippet?

File: content/modules/r/pages/merge.adoc [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+
+but now has extra columns, namely, with information about the `Origin` airport:
+
+[source, R]
+----
+head(mynewDF)
+tail(mynewDF)
+----
```
Comment (lacykq) [inline content/modules/r/pages/merge.adoc:117]: Add the output for the code snippet?

File: content/modules/r/pages/merge.adoc [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+
+So now we can do things like calculating a `sum` of all `Distances` of flights with `Origin` in each `state`:
+
+[source, R]
+----
+sort(tapply( mynewDF$Distance, mynewDF$state, sum ))
```
Comment (lacykq) [inline content/modules/r/pages/merge.adoc:123]: Output?

File: content/modules/r/pages/merge.adoc [hunk 6]
```diff
@@ -0,0 +1,302 @@
...
+----
+
+== Here is another `merge` example:
+
+=== Examples
```
Comment (lacykq) [inline content/modules/r/pages/merge.adoc:128]: Delete

File: content/modules/r/pages/merge.adoc [hunk 7]
```diff
@@ -0,0 +1,302 @@
...
+
+Consider the data.frame's `books` and `authors`:
```
Comment (lacykq) [inline content/modules/r/pages/merge.adoc:130]: Consider the data.frames, `books` and `authors`:

File: content/modules/r/pages/merge.adoc [hunk 8]
```diff
@@ -0,0 +1,302 @@
...
+4   4            Robert Jordan       4.18
+5   5          Agatha Christie       4.00
+6   6                Alex Kava       4.02
+7   7    Nassim Nicholas Taleb       3.99
+8   8              Neil Gaiman       4.13
+9   9            Stieg Larsson       4.16
+10 10 Antoine de Saint-Exupéry       4.30
+----
+
+==== How do I merge the author information from `authors` based on `author_id` in `books` and `id` in `authors`, keeping only information from authors and books where there is a match?
```
Comment (lacykq) [inline content/modules/r/pages/merge.adoc:172]: How do I merge both data.frames based on the `author_id` column in the `books` data and the `id` column in the `authors` data, keeping only rows where there's a match?

File: content/modules/r/pages/merge.adoc [hunk 9]
```diff
@@ -0,0 +1,302 @@
...
+5       J.K. Rowling       4.46
+6       J.K. Rowling       4.46
+7       J.K. Rowling       4.46
+8  Brandon Sanderson       4.39
+9       Markus Zusak       4.34
+10     Robert Jordan       4.18
+----
+====
+
+==== How do I merge the author information from `authors` based on `author_id` in `books` and `id` in `authors`, keeping all information from `authors` regardless of whether or not there is match?
```
Comment (lacykq) [inline content/modules/r/pages/merge.adoc:210]: How do I merge both data.frames based on the `author_id` column in the `books` data and the `id` column in the `authors` data, keeping **all** rows regardless of whether or not there is a match?

**Comment by dglass19** [comment]
I think we still need to add output for these code snippets, but to clean up the PR I'm going to go ahead and merge it. I'll add a banner about changes.

**Pull request merged**